resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,124 @@
+openapi: "3.0.3"
+info:
+  title: API Platform Test
+  version: 2.0.0
+  contact: {}
+servers:
+  - url: https://test-api.service.hmrc.gov.uk/test/api-platform-test
+    description: Sandbox
+  - url: https://api.service.hmrc.gov.uk/test/api-platform-test
+    description: Production
+    variables: {}
+
+components:
+  parameters:
+    acceptHeader:
+      name: Accept
+      in: header
+      schema:
+        type: string
+        enum: [
+          "application/vnd.hmrc.2.0+json"
+        ]
+      required: true
+paths:
+  /:
+    get:
+      summary: dummy resource
+      description: |
+        dummy resource
+        This endpoint is open access and requires no Authorization header.
+      tags:
+        - api-platform-test
+      parameters:
+        - $ref: '#/components/parameters/acceptHeader'
+      responses:
+        200:
+          description: "OK Response"
+      security:
+        - {}
+  /ciao:
+    get:
+      summary: dummy Ciao resource
+      description: |
+        dummy Ciao resource
+        This endpoint is open access and requires no Authorization header.
+      tags:
+        - api-platform-test
+      parameters:
+        - $ref: '#/components/parameters/acceptHeader'
+        - name: surname
+          in: query
+          description: |
+            surname
+          required: false
+          schema:
+            type: string
+            example: smith
+        - name: firstName
+          in: query
+          description: |
+            first name
+          required: false
+          schema:
+            type: string
+            example: john
+        - name: middleName
+          in: query
+          description: |
+            first name
+          required: false
+          schema:
+            type: string
+            example: henry
+      responses:
+        200:
+          description: "OK Response"
+      security:
+        - {}
+  /{surname}:
```

Comment:
is this at the same level as the firstname one.  Appreciate this is just mock stuff but it looks wrong.

Reply:
Good spot. Fixed